webapps/assembly/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,32 @@
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <logger name="org.apache.ibatis" level="info" />
+
+  <logger name="javax.activation" level="info" />
+
+  <logger name="org.springframework" level="info" />
+
+  <!-- wiremock logger -->
+  <logger name="org.mortbay.log" level="info" />
```

Comment:
🔧 The wiremock is not used in webapps tests. 
🔧 I suggest to add the jetty logger as there are a lot of output from it:
```suggestion
  <logger name="org.eclipse.jetty" level="info" />
```